webcompat/templates/home-page/form.html

```diff
@@ -85,7 +85,7 @@ <h1 class="headline-1">Report Site Issue</h1>
         {{ subradio.label(class_='form-label label-radio') }}
       {% endfor %}
     </div>
-    <div class="form-upload js-image-upload js-Form-group">
+    <div class="form-upload form-element js-image-upload js-Form-group">
```

Comment:
What does `.form-element` do for us here?

Reply:
@miketaylr probably structural spacing if I remember? but the new design kind of messed up my own understanding of the form structure.

Reply:
Maybe/Probably @magsout knows better

Reply:
Not a blocker, I was just curious.

Reply:
@miketaylr @karlcow it is used for making space between differents form elements `margin: 0 0 20px`.

Reply:
![capture d ecran 2018-12-04 a 09 32 55](https://user-images.githubusercontent.com/1997108/49448785-99fe9e80-f7a7-11e8-9bdd-96a13be6de50.png)